docs/wiki/deployment/yara.md

```diff
@@ -0,0 +1,138 @@
+# Introduction
+
+I recently put [YARA](https://github.com/plusvic/yara) inside [osquery](https://osquery.io) and thought I would provide some details on how to use it. There are two YARA related tables in osquery, which serve very different purposes. The first table, called **yara_events**, uses osquery's pub-sub framework to monitor for filesystem changes and will execute YARA when a file change event fires. The second table, called **yara**, is an on-demand YARA scanning table.
```

Comment:
I would cut this whole first sentence. The docs should read as third-person reference, rather than first-person.